doc/nrf/ug_thingy53_gs.rst

```diff
@@ -0,0 +1,274 @@
+.. _ug_thingy53_gs:
+
+Getting started with Thingy:53
+##############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide helps you get started with the Nordic Thingy:53.
+It tells you how to update the firmware on the nRF5340 :term:`System on Chip (SoC)` on the Nordic Thingy:53, and how to get started with machine learning with Edge Impulse.
+
+If you have already set up your Nordic Thingy:53 and want to learn more, see the following documentation:
+
+* :ref:`ug_thingy53` for more advanced topics related to the Nordic Thingy:53.
+* The :ref:`introductory documentation <getting_started>` for more information on the |NCS| and the development environment.
+
+.. _thingy53_gs_requirements:
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer or mobile device has one of the supported operating systems.
+
+Hardware
+========
+
+* :term:`Universal Serial Bus (USB)`-C cable
+* Computer or mobile device
+
+Software
+========
+
+If you are using a computer, one of the following operating systems:
+
+* Microsoft Windows 8 or 10
+* macOS, latest version
+* Ubuntu Linux, latest Long Term Support (LTS) version
+
+If you are using a mobile device, one of the following operating systems:
+
+* Android
+* iOS
+
+.. _thingy53_gs_installing_software:
+
+Installing the required software
+********************************
+
+When using a computer to work with with the Nordic Thingy:53 firmware, install `nRF Connect for Desktop`_.
+After installing and starting the application, install the Programmer app.
+
+When using a mobile device to work with the Nordic Thingy:53, install the `nRF Programmer`_ mobile application from the corresponding application store.
+
+.. _thingy53_gs_updating_firmware:
+.. _thingy53_gs_precompiled_firmware:
+
+Getting started with precompiled firmware samples
+*************************************************
+
+The Nordic Thingy:53 is preloaded with Edge Impulse firmware, application firmware, and a SoftDevice controller on the network core of the nRF5340 SoC.
+For more information on the firmware and associated features, see :ref:`ug_thingy53`.
```

Comment:
Shouldn't we link to Thingy:53's EI page here?

Reply:
They don't have one, at least at the moment. That's why the instructions in the Getting started with machine learning section refer to the instructions for nRF5340 DK.